CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## Important Notes
 
+- [#696](https://github.com/oauth2-proxy/oauth2-proxy/pull/696) Preserve query when building redirect
```

Comment:
Move this line down to the `## Changes since v6.0.0` section (starting on line 13)

Reply:
Oops, yeah that makes more sense. Fixed in latest update. Thanks!